python_modules/libraries/dagster-dg/dagster_dg_tests/utils.py

```diff
@@ -91,6 +91,27 @@ def isolated_components_venv(runner: Union[CliRunner, "ProxyRunner"]) -> Iterato
             yield venv_path
 
 
+@contextmanager
+def isolated_dg_venv(runner: Union[CliRunner, "ProxyRunner"]) -> Iterator[Path]:
+    with runner.isolated_filesystem():
+        subprocess.run(["uv", "venv", ".venv"], check=True)
```

Comment:
I think this is a good case to do it clean , but we need to cut back on how often we are doing costly fresh venvs in the test suite to get it back down to a reasonable run time

Reply:
agree